.github/workflow/aggregate.yaml

```diff
@@ -0,0 +1,44 @@
+
+name: Aggregate daily data
+on: 
+  schedule:
+    # at 12AM, 6AM, 12PM, and  6PM
+    - cron '0 0/6 * * *'
+
+jobs:
+  test-and-build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set NodeJS version
+        id: node_version
+        run: echo ::set-output name=NODE_VERSION::$(cat .nvmrc)
+
+      - name: Set Up NodeJS
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ steps.node_version.outputs.NODE_VERSION }}
+          cache: "yarn"
+
+      - name: Setup dependencies
+        run: yarn install --immutable --check-cache
+        run: yarn add ardrive-cli
+
+      - name: Write wallet
+        run: echo -n '${{ secrets.WALLET_FILE }}' | base64 --decode > /tmp/wallet
+
+      - name: Lint
+        run: yarn lint
+
+      - name: Build
+        run: yarn build
+
+
+      - name: Fetch previous output
+        run: ardrive download-file -f a7afa47b-a16c-44ad-8d87-c63f935af568
```

Comment:
from where you got this file-id? I made an upload before of the latest json that I generated locally and the file-id is 7fa5d4e3-0087-422a-acb3-2e481d98d08b

Reply:
```suggestion
        run: ardrive download-file -f 7fa5d4e3-0087-422a-acb3-2e481d98d08b
```